setup.py

```diff
@@ -432,7 +432,7 @@ def run(self):
         extra_compile_args.append("/Z7")
         extra_link_args.append("/DEBUG")
 else:
-    extra_compile_args = ["-Werror"]
```

Comment:
Is this change related?

Reply:
No, this was an accidental commit. You currently cannot develop on OSX with `Werror` on as you get a long list of warning about unused linker flags. I have to manually apply this every time I do something with `pandas`, raised that already a long time ago in https://github.com/pandas-dev/pandas/issues/33315